ffv1.md

```diff
@@ -133,8 +133,11 @@ median(a,b,c)         the numerical middle value in a data set of a, b, and c, i
 
 a~b~                  the b-th value of a sequence of a
 
-a~b,c.                the 'b,c'-th value of a sequence of a
+a~b,c~                the c-th value of a sequence of a~b~
```

Comment:
Maybe some additional text, an example would help here.

Reply:
updated the pull req, removing the whole sequence stuff (that is all uses of ~)